pkg/auth/artifact_verifier_test.go

```diff
@@ -45,7 +46,13 @@ func testVerifiedWithFiles(t *testing.T, files []testFile, verifier ArtifactVeri
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = verifier.VerifyHoistArtifact(localCopy, fmt.Sprintf("file://%v", filePath))
+
+	url, err := url.Parse(fmt.Sprintf("file://%v", filePath))
```

Comment:
I did it this way to ensure that the behavior of the test doesn't change, but given that they pass now it should be okay to do what you're suggesting